templates/takeovers/_improving_productivity_kubernetes.html

```diff
@@ -0,0 +1,35 @@
+<section class="p-strip--image is-dark is-deep u-image-position js-takeover p-improving-prod-takeover">
+      <div class="row">
+        <div class="col-6" >
+          <h1 style="font-weight: 100;" class="u-no-margin--bottom u-sv3">Improving productivity with Ubuntu Kubernetes</h1>
+          <p class="p-heading--four">Canonical helped investment company, Allan Gray, deploy its Ubuntu Kubernetes cluster in under 2 weeks.</p>
+          <p class="u-hide--small"><a href="https://www.ubuntu.com/engage/kubernetes-case-study?utm_source=takeover&utm_medium=takeover&utm_campaign=2)FY19_Cloud_K8_CStudy_AllanGray" class="p-button--neutral u-no-margin--bottom" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Improving Productivity - Kubernetes takeover', 'eventLabel' : 'Download the case study', 'eventValue' : undefined });">Download the case study</a></p>
+        </div>
+        <div class="col-6 u-vertically-center">
+          <div class="u-align--center">
+            <img src="{{ ASSET_SERVER_URL }}1db850da-K8s+logo.svg" alt="Kubernetes logo" class="p-improv-k8-takeoever-image"/>
+          </div>
+          <p class="u-hide--large u-hide--medium"><a href="https://www.ubuntu.com/engage/kubernetes-case-study?utm_source=takeover&utm_medium=takeover&utm_campaign=2)FY19_Cloud_K8_CStudy_AllanGray" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Improving Productivity - Kubernetes takeover', 'eventLabel' : 'Download the case study', 'eventValue' : undefined });">Download the case study</a></p>
+        </div>
+      </div>
+  <style>
+    .p-improv-k8-takeoever-image{
```

Comment:
Please give takeovers the same block name. This should be `p-improving-prod-takeover__image`